commands/MustangCommand.java

```diff
@@ -20,10 +20,17 @@ public interface MustangCommand extends Command {
     public Map<MustangSubsystemBase, MustangSubsystemBase.HealthState> getHealthRequirements();
 
     /**
-     * scheuldes this command
+     * logs data revelant to debugging this command
+     * For use when execute or other parts of command have logical errors
+     */
+    public void debugCommand();
```

Comment:
In the beginning, it seemed like a valuable addition in case our planned telemetry changes did not work out, but now that we have a working advantage kit implementation, it will be removed soon